source/PreferencesPanel.cpp

```diff
@@ -140,6 +140,8 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		++currentSettingsPage;
 	else if((key == 'r' || key == SDLK_PAGEDOWN) && currentSettingsPage > 0)
 		--currentSettingsPage;
+	else if(key == 'x' || key == SDLK_DELETE)
+		Command::SetKey(zones[latest].Value(), 0);
```

Comment:
This doesnt prevent assigning a key to 0, does it?

Reply:
IIRC It's the SDL_Keycode 0, which is undefined and not on a keyboard, so cannot be assigned otherwise.

Reply:
We can still give a value to 'x', right? How does that work?

Reply:
https://wiki.libsdl.org/SDL2/SDLKeycodeLookup

Here at 0x00

Reply:
Oh nvm I get it this works when you've unselected the keybind which is why we need to store the latest one.